pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -230,6 +230,8 @@ stdenv.mkDerivation ({
     # replicated here to apply to older versions.
     # Makes __FILE__ relative to the build directory.
     "KCPPFLAGS=-fmacro-prefix-map=$(sourceRoot)/="
+  ] ++ lib.optionals (kernelConf?target) [
+    "KBUILD_IMAGE=${kernelConf.target}"
```

Comment:
Ah yes, great.  I've been wanting to do this for a while, but haven't had a reason because (at least for aarch64) it doesn't work for the really old kernels.  For POWER/MIPS, maybe that's fine.

Reply:
It doesn't work there either.  I can't get this to work and I've run out of time.